jira-doc.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import print_function
+from collections import Counter
+from jira import JIRA
+from datetime import datetime, timedelta
+import json
+import ast
+import re
+
+def get_list(sprint):
+    query_string = ('project=JD and status="Done" and Sprint="' + sprint +'"')
+    old_sprint = (int(sprint)-1)
+    query_strin_old_sprint = ('project=JD and status="Done" and Sprint="' + str(old_sprint) +'"')
+    done = jira.search_issues(query_string)
+    done_old_sprint = jira.search_issues(query_strin_old_sprint)
+    print ('Done')
+    file = open("Report-"+str(datetime.now())+".txt", "w")
+    file.write('Done \n')
+    for x in done:
+        if x.raw['fields']['assignee']is None:
+            output = (x.raw['fields']['summary']+'.')
+
+            file.write(output.encode('utf-8') + '\n')            
+        else:            
+            output = (x.raw['fields']['summary'] +' '+ x.raw['fields']['assignee']['displayName']+'.')
+           
+            file.write(output.encode('utf-8') + '\n')
+
+    for x in done_old_sprint:
+        update_date = x.raw['fields']['updated']
+        update_date = update_date.split('.', 1)[0]
+        then = datetime.strptime(update_date, "%Y-%m-%dT%H:%M:%S")
+        # print(then > datetime.now() - timedelta(days = 7))
+        # print(then)
+        # print(datetime.now() - timedelta(days = 7))
+        if then > datetime.now() - timedelta(days = 7):
+            if x.raw['fields']['assignee']is None:
+                output = (x.raw['fields']['summary'])
+
+                file.write(output.encode('utf-8') + '\n')
+            else:            
+                output = (x.raw['fields']['summary'] +' '+ x.raw['fields']['assignee']['displayName'])
+
+                file.write(output.encode('utf-8') + '\n')
+
+    query_string = ('project=JD and status="In progress" and Sprint="' + sprint +'"')
+    inprogress = jira.search_issues(query_string)
+    print ('In progress')
+    file.write('In progress \n')
+    for x in inprogress:
```

Comment:
remove repetition of these loops (todo, in progress, done), extract them to a separate function and call with parameters.